src/impls.rs

```diff
@@ -108,14 +105,14 @@ impl<'a> Drop for RWIobuf<'a> {
 /// 0xFF, and consuming/validating these numbers in parallel in 4 other threads:
 ///
 /// ```rust
+/// #![allow(unstable)]
```

Comment:
Does this example still need #[allow(unstable)], even with the range_inclusive change?